src/main/java/com/thealgorithms/backtracking/NQueens.java

```diff
@@ -19,10 +19,10 @@
  *
  * Generate all possible arrangement to place N queens on N*N board. Check each
  * board if queens are placed safely. If it is safe, include arrangement in
- * solution set. Otherwise ignore it
- *
+ * solution set. Otherwise, ignore it
+ * <p>
```

Comment:
No need for paragraphs, they complicate reading this in code. Is there a benefit to having them? Would a <br/> work instead?

Reply:
i will remove it